themes/default/content/what-is/what-are-docker-secrets.md

```diff
@@ -0,0 +1,168 @@
+---
+title: What are Docker Secrets?
+meta_desc: |
+     Learn more about what Docker Secrets are and how to use them.
+
+type: what-is
+page_title: "What are Docker Secrets"
+
+customer_logos:
+  title: Leading engineering organizations are building with Pulumi
+  logos:
+    - items:
+      - snowflake
+      - tableau
+      - atlassian
+      - fauna
+      - ware2go
+    - items:
+      - mindbody
+      - sourcegraph
+      - fenergo
+      - skai
+      - lemonade
+    - items:
+      - clearsale
+      - angellist
+      - webflow
+      - supabase
+      - ro
+---
+
+Docker, a leading platform in containerization technology, has revolutionized how applications are developed, shipped, and deployed. One critical aspect of this process is managing sensitive information, commonly known as "secrets." [Docker Secrets](https://docs.docker.com/engine/swarm/secrets/) is a feature specifically designed for safely transmitting and storing confidential data within Docker environments. This guide explores the ins and outs of Docker Secrets, highlighting its importance, functionality, and best practices.
+
+### What are Docker Secrets?
+
+Docker Secrets is a resource for securely managing sensitive data like passwords, tokens, and SSH keys within [Docker Swarm](https://docs.docker.com/engine/swarm/) environments. It's designed to keep this information safe both in transit and at rest.
+
+#### Key Features
+
+- **Secure Storage**: Docker Secrets are encrypted during transit and at rest, offering a robust level of security.
+- **Managed Lifecycle**: Secrets can be created, updated, and removed without restarting containers, ensuring seamless management.
+- **Access Control**: Only services granted explicit access can retrieve these secrets, enhancing security through compartmentalization.
+
+### Creating Secrets
+
+Secrets can be created via the Docker CLI or Docker Compose files. Once created, they are stored in a secure part of the Docker Swarm. Before creating secrets in Docker, you must first make sure you have [Docker installed](https://docs.docker.com/get-docker/). Once you have installed Docker, enable and start the Docker service.
+
```

Comment:
It may be worthwhile to add a quick note here indicating that these initial instructions are for Linux. (Users running Docker Desktop for Mac, for example, wouldn't need to run `systemctl`.)

Reply:
Updated.